packages/js/src/ui/components/Renderer.tsx

```diff
@@ -75,13 +79,14 @@ export const Renderer = (props: RendererProps) => {
           <FocusManagerProvider>
             <InboxProvider tabs={props.tabs}>
               <CountProvider>
-                <For each={[...props.nodes]}>
-                  {([node, component]) => {
+                <For each={nodes()}>
```

Comment:
The nodes aka DOM element refs don't change on re-render, so it's "safe" to use them in the `For` component `each` prop as the Solid is doing comparison based on refs when rendering DOM elements. This code will make sure that the corresponding Solid components won't remount on props change.